config/language_server_node_eslint.yml

```diff
@@ -0,0 +1,11 @@
+# TODO License.
+
+# Configuration for the JAVA LSP server component
+---
+version: 0.0.+
+repository_root: "https://lsp-component-eslint.cfapps.sap.hana.ondemand.com"
```

Comment:
I'm not able to access https://lsp-component-eslint.cfapps.sap.hana.ondemand.com/index.yml